unstract/task-abstraction/README.md

```
@@ -0,0 +1,115 @@
+
+Set the backend type via environment variable:
+
+```bash
+export TASK_BACKEND_TYPE=celery
+export TASK_CELERY_BROKER_URL=redis://localhost:6379/0
+export TASK_CELERY_RESULT_BACKEND=redis://localhost:6379/0
+```
+
+## Documentation
+
+For complete documentation, visit: https://docs.unstract.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)